build.gradle

```diff
@@ -2,22 +2,19 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        // org.jetbrains.trove4j:trove4j:20160824.
```

Comment:
nit: What's this comment for?

Reply:
Ah, I was marking what artifact we need from that repo. I guess we don't necessarily need the comment. 🤷 

I'll remove it. 👍